README.md

```diff
@@ -72,8 +72,8 @@ one the "Code" button. Or use the following command:
 * flex
 * libreadline
 * openssl development library (for OVIS, LDMS Authentication)
-* libmunge (for Munge LDMS Authentication plugin)
-* Python >= 3.6 and Cython >= 0.25 (for the LDMS Python API and ldmsd_controller)
+* libmunge development library (for Munge LDMS Authentication plugin)
+* Python >= 3.6 development library and Cython >= 0.25 (for the LDMS Python API and the LDMSD Interface, ldmsd_controller)
```

Comment:
I think we rely on 0.29 for Cython now., but the issue that prompted this was that the user in question didn't read any of this, they just went directly to the Building section which doesn't reference these dependencies. So I think we need a pointer in the "Building" section that refers back to the dependencies section.

Reply:
@narategithub could you please collect the versions from the build container that we have? I know it's a pain, but we know that works. Also this readme should refer to that container and how one builds using it.